wallet/res/layout/dialog_change_tax_category_explainer.xml

```diff
@@ -0,0 +1,100 @@
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    tools:orientation="vertical"
+    tools:context="de.schildbach.wallet.ui.transactions.ChangeTaxCategoryExplainerDialogFragment"
+    tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
+
+    <ImageButton
+        android:id="@+id/collapse_button"
+        style="@style/DialogCloseButton"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/title"
+        style="@style/Headline6.Bold"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="20dp"
+        android:layout_marginTop="20dp"
+        android:text="@string/reclassify_transactions_question"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/collapse_button" />
+
+    <TextView
+        android:id="@+id/explanation"
+        style="@style/Body2"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="20dp"
+        android:layout_marginTop="10dp"
+        android:text="@string/reclassify_transactions_change_explanation"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/title" />
+
+    <FrameLayout
+        android:id="@+id/frame"
+        android:layout_width="match_parent"
+        android:layout_height="360dp"
+        android:layout_marginHorizontal="20dp"
+        android:layout_marginTop="30dp"
+        android:background="@drawable/rounded_corners_bgd_light_gray"
+        app:layout_constraintTop_toBottomOf="@+id/explanation"
+
+        />
+
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/transaction_details"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/white_background_rounded"
+        android:layout_marginBottom="-40dp"
+        android:scaleX="0.667"
+        android:scaleY="0.667"
+        app:layout_constraintTop_toTopOf="@+id/frame">
+
+        <include
+            layout="@layout/transaction_result_content" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
here we insert the transaction details layout scaled down to 67%.  The background is set to a rounded white rectangle.

Reply:
I was able to achieve a closer to the design look by using `translationY`. Although it might require some more massaging of the margins and checking different screen sizes. The container is put inside the `FrameLayout` to have the bottom cut-off:
```
<FrameLayout
        android:id="@+id/frame"
        android:layout_width="match_parent"
        android:layout_height="0dp"
        android:layout_marginTop="30dp"
        android:layout_marginBottom="54dp"
        android:layout_marginHorizontal="20dp"
        android:background="@drawable/rounded_corners_bgd_light_gray"
        app:layout_constraintTop_toBottomOf="@+id/explanation"
        app:layout_constraintBottom_toBottomOf="parent">

        <androidx.constraintlayout.widget.ConstraintLayout
            android:id="@+id/transaction_details"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:translationY="50dp"
            android:layout_marginVertical="-50dp"
            android:scaleX="0.77"
            android:scaleY="0.77"
            android:background="@drawable/white_background_rounded">

            <include
                layout="@layout/transaction_result_content" />

        </androidx.constraintlayout.widget.ConstraintLayout>
    </FrameLayout>
```

Reply:
This works very well.